package.json

```diff
@@ -10,6 +10,7 @@
     "typescript": "tsd",
     "typescript:jest": "jest",
     "typescript:esm": "node scripts/unit-typescript-esm.js",
+    "typescript:swc": "node scripts/unit-typescript-swc.js",
```

Comment:
you need to add this command to `test` script to be handled by the ci. Otherwise lgtm

Reply:
oh good call, thanks!